code/game/gamemodes/miniantags/demons/slaughter demon/slaughter.dm

```diff
@@ -72,6 +72,7 @@
 /mob/living/simple_animal/demon/slaughter/Destroy()
 	// Only execute the below if we successfully died
 	for(var/mob/living/M in consumed_mobs)
+		REMOVE_TRAIT(M, TRAIT_UNREVIVABLE, "unrevivable")
```

Comment:
```suggestion
		REMOVE_TRAIT(M, TRAIT_UNREVIVABLE, "demon")
```
Gotta also change this one. Else you won't actually remove the trait since you're removing it with a different source

Reply:
👏 Ah I forgot to look for that one